web/src/hotkey.js

```diff
@@ -382,9 +382,7 @@ export function process_escape_key(e) {
         return true;
     }
 
-    /* The Ctrl+[ hotkey navigates to the home view
-     * unconditionally; Esc's behavior depends on a setting. */
-    if (user_settings.web_escape_navigates_to_home_view || e.which === 219) {
+    if (user_settings.web_escape_navigates_to_home_view) {
```

Comment:
This commit is I think wrong; we code `Ctrl+[` to call the same function as `Esc`, but this detail behaves differently. 

```
const keydown_ctrl_mappings = {                                                                         
    219: {name: "escape", message_view_only: false}, // '['                                             
    13: {name: "ctrl_enter", message_view_only: true}, // enter                                         
};                                                                                                      
```